File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/Organization.ts
import { Address } from "./Address";
import { CodeableConcept } from "./CodeableConcept";
import { ContactPoint } from "./ContactPoint";
import { HumanName } from "./HumanName";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";
import { BackboneElement } from "./BackboneElement";

export type OrganizationContact = BackboneElement & {
    name?: HumanName;
    address?: Address;
    purpose?: CodeableConcept;
    telecom?: ContactPoint[];
};

export type Organization = DomainResource & {
    address?: Address[];
    name?: string;
    type?: CodeableConcept[];
    alias?: string[];
    active?: boolean;
    identifier?: Identifier[];
    telecom?: ContactPoint[];
    partOf?: Reference;
    endpoint?: Reference[];
    contact?: OrganizationContact[];
};

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/Organization.py
from typing import Optional, List
from base import Address
from base import CodeableConcept
from base import ContactPoint
from base import HumanName
from base import DomainResource
from base import Reference
from base import Identifier
from base import BackboneElement

class Organization_Contact(BackboneElement):
    name: Optional[HumanName] = None
    address: Optional[Address] = None
    purpose: Optional[CodeableConcept] = None
    telecom: Optional[List[ContactPoint]] = None

class Organization(DomainResource):
    address: Optional[List[Address]] = None
    name: Optional[str] = None
    type: Optional[List[CodeableConcept]] = None
    alias: Optional[List[str]] = None
    active: Optional[bool] = None
    identifier: Optional[List[Identifier]] = None
    telecom: Optional[List[ContactPoint]] = None
    part_of: Optional[Reference] = None
    endpoint: Optional[List[Reference]] = None
    contact: Optional[List[Organization_Contact]] = None